_posts/2015-11-06-try-new-roi-features-on-the-demo-app.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "Try new ROI features on the demo app"
+date:   2015-11-06
+---
+
+As you may have heard, we are currently adding ROI support to OMERO.figure, allowing
+you to draw ROIs on image panels and copy & paste these between images.
+This post describes the new features, invites you to try them out on the demo app
+and provide feedback ahead of their inclusion in the next release. 
+
+<h2>Adding ROIs to image panels</h2>
+
+<img src="https://cloud.githubusercontent.com/assets/900055/9478881/5894a230-4b72-11e5-91c7-6ac2237a0a67.png" 
+  style="width:600px"/>
+
+To add ROIs to panels in the figure, select a single panel, click on the "Labels" tab
+and then click on the "Draw" button under the ROIs header.
+This will launch the ROI viewer, where you can draw, edit and delete a number of ROI
```

Comment:
You do mention edit and delete here but the section title is Adding ROIs and given the button is labelled "Draw" I wonder if either:
 - the section title could be expanded
 - the editting and deleting could be highlighted - perhaps a small extra section?
- or, this is more about the code, could there be a better label for the Draw button?